components/Main/Exercises/Exercises.js

```diff
@@ -26,44 +26,92 @@ export default function Exercises() {
 
   return (
     <StyledExercises>
-      <h2>Biceps Exercises</h2>
-      <SearchBar onChange={handleChange} setSearchTerm={setSearchTerm} />
-      {dataBiceps
-        .filter((exercise) => {
-          if (searchTerm === "") {
-            return exercise;
-          } else if (
-            exercise.name.toLowerCase().includes(searchTerm.toLowerCase())
-          ) {
-            return exercise;
-          }
-        })
-        .map(({ name, equipment, difficulty, instructions, id }) => (
-          <li key={id}>
-            <div className="favContainer">
-              <button
-                onClick={() => {
-                  handleToggle(id);
-                }}
-              >
-                <span>
-                  <h3>{name.toUpperCase()}</h3>
-                </span>
-              </button>
-              <Favorite />
-            </div>
-            {toggledExercises.includes(id) ? (
-              <Description
-                key={id}
-                id={id}
-                name={name}
-                equipment={equipment}
-                difficulty={difficulty}
-                instructions={instructions}
-              />
-            ) : null}
-          </li>
-        ))}
+      {!isFavouriteExercise ? (
+        <>
+          <h2>Biceps Exercises</h2>
+          <SearchBar onChange={handleChange} setSearchTerm={setSearchTerm} />
+          {dataBiceps
+            .filter((exercise) => {
+              if (searchTerm === "") {
+                return exercise;
+              } else if (
+                exercise.name.toLowerCase().includes(searchTerm.toLowerCase())
+              ) {
+                return exercise;
+              }
+            })
```

Comment:
Maybe you could put the searchbar into a separate components so that this code gets a little more simple and clear? Not necessary but if you have the time and want to do it, it could be a way to clean up your code :)

Reply:
Thanks! I had the same thought. I'm gonna do it on a refactoring day. Spent too much time with the whole add fav thing. :D